webviews/components/comment.tsx

```diff
@@ -353,13 +353,12 @@ export function AddComment({
 						{continueOnGitHub ? 'Approve on github.com' : 'Approve'}
 					</button>
 				) : null}
-				<input
+				<button
 					id="reply"
-					value="Comment"
 					type="submit"
-					className="secondary"
```

Comment:
@alexr00 Not sure I have the full context here—we're these styled secondary for some reason?

Before/after screenshots would be helpful here and for all other UI changes @Thomas1664

Reply:
I don't actually see a `primary` class in the css files.

Reply:
Styling it as `secondary` appears to have just been to get the alignment correct: https://github.com/microsoft/vscode-pull-request-github/pull/1350